app/services/npq/application/accept.rb

```diff
@@ -24,6 +27,9 @@ def call
           npq_application.update!(lead_provider_approval_status: "accepted")
           other_applications_in_same_cohort.update(lead_provider_approval_status: "rejected") # rubocop:disable Rails/SaveBang
           deduplicate_by_trn!
+          if FeatureFlag.active?("npq_capping")
```

Comment:
can we add the check that if funded place is set as well, incase it's in previous cohorts?